setup.py

```diff
@@ -41,6 +41,8 @@
         "ci_sdr",
         "pytorch_wpe",
         "fast-bss-eval==0.1.3",
+        # SPK
+        "asteroid_filterbanks==0.4.0",
```

Comment:
do we need this?
is it possible to use our own fiterbanks?

Reply:
Oh, this is not actually filterbanks. It's a type of layer that processes raw waveforms (similar to sinc-conv layer in SincNet)
Let me know if I have to remove it.

Reply:
Ok!